clients/source.go

```diff
@@ -138,6 +141,7 @@ func (c *SourceClient) newManagedClient(ctx context.Context, path string) (*Sour
 				jsonToLog(c.logger, structuredLogLine)
 			}
 		}
+		c.wg.Done()
```

Comment:
Use `defer c.wg.Done()` (start of func) in case of panics.

Reply:
Good point, fixed 👍